src/components/Footer.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import styled from 'styled-components'
+import logo from "../images/cfc_logo_short.svg"
+
+const StyledFooter = styled.footer`
+width: 14rem;
+`
+const Footer = () => {
+  return (
+    <StyledFooter>
+      <img src={logo} alt="Code for Chicago Logo Icon"/>
+      <p>
+        <a href="https://codeforamerica.org/" target="_blank" rel="noreferrer">Code for America Labs, Inc.</a> is a non-partisan, non-political 501(c)(3) charitable organization</p>
```

Comment:
Looks like you my not have prettier on for this project. It should have formatted it a bit like this. Would you do me a favor and double check that prettier is working as expected? 
<img width="868" alt="Screen Shot 2022-11-19 at 6 21 44 PM" src="https://user-images.githubusercontent.com/6326891/202876826-eed8c788-a382-4f05-936a-55c1528ffd1d.png">

Reply:
Ok, all fixed. Still getting the hang of Prettier